### PR TITLE
ci: add nftables-master to test matrix

### PR DESCRIPTION
**Pull request by erig0**
The testsuite often catches regressions in nftables. Lets always run
against nftables master to catch regression earlier.

### CODE REVIEW

**Comment by erig0** [comment]
@todoleza, tagging you only as FYI.

File: src/tests/functions.at [hunk 1]
```diff
@@ -392,6 +392,7 @@ m4_define([NFT_LIST_RULES_NORMALIZE], [dnl
         -e '/type.*hook.*priority.*policy.*/d'dnl
         dnl tranform ct state { established,related } to ct state established,related
         -e '/ct \(state\|status\)/{s/\(ct \(state\|status\)\) {/\1/g; s/ }//; s/\(@<:@a-z@:>@*\), /\1,/g;}' dnl
+        -e 's/reject with icmp[[x6]]\? type port-unreachable/reject/' dnl
```
Comment (n0-1) [inline src/tests/functions.at:395]: ICMPv6 syntax is: 'reject with icmpv6 ...'. So the match part should read: 'reject with icmp\(x\|v6\)\? type port-unreachable'.

Comment (erig0) [reply src/tests/functions.at:395]: Thanks for catching that. I'll push a follow up.

Comment (erig0) [reply src/tests/functions.at:395]: I directly pushed commit 3a3b4676ccb7b40cf304b773456dec2662783425.

**Pull request merged**